src/sentry/api/serializers/models/team.py

```diff
@@ -168,14 +167,14 @@ class TeamSerializerResponse(_TeamSerializerResponseOptional):
     name: str
     dateCreated: datetime
     isMember: bool
-    teamRole: str
-    flags: dict[str, Any]
-    access: frozenset[str]  # scopes granted by teamRole
+    teamRole: Optional[str]
```

Comment:
Change teamRole to Optional b/c our API can still return these as null. When we serialize a team, we're always passing in None if teamRole does not exist so this will always be set regardless